tests/e2e/requirements.txt

```diff
@@ -9,5 +9,5 @@ pytest-timeout==1.3.3
 pytest-ordering==0.6.*
 tensorflow-data-validation==0.21.2
 deepdiff==4.3.2
-tensorflow==2.1.0
+tensorflow==2.2.0
```

Comment:
This should probably not have been changed right?

Reply:
will be reverted, It was an effort to fix dependency error that appears in end-to-end tests.